### ISSUE TITLE
Is rplidar A1 possible to change direction when I set up on the robot?

### ISSUE THREAD

**Issue by ChangkiSung** [Slamtec/rplidar_ros#7]
Hi, I am trying to use rplidar a1 on my robot. 

It has default front direction in ROS, but I would like to change the direction (Rotate 90 degrees to clockwise). Is that possible? If it is possible, what should I change??

Thank you.

**Comment by salihmarangoz** [comment]
This is wrong. It is sensor's default value. If you mounted laser with 90 degree rotated, you have to publish it's transform as rotated. This information must be published in base_link -> laser transform.
